policy/scan/local_scanner.go

```diff
@@ -265,10 +249,11 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstream *up
 		for _, a := range runtime.Provider.Connection.GetInventory().GetSpec().GetAssets() {
 			a.AddMondooLabels(resolvedAsset)
 			a.AddAnnotations(resolvedAsset.GetAnnotations())
+			a.ManagedBy = resolvedAsset.ManagedBy
```

Comment:
This line is the actual fix. The rest of the PR is refactoring to allow adding a test case.